django/db/models/functions/__init__.py

```diff
@@ -25,4 +26,6 @@
     # window
     'CumeDist', 'DenseRank', 'FirstValue', 'Lag', 'LastValue', 'Lead',
     'NthValue', 'Ntile', 'PercentRank', 'Rank', 'RowNumber',
+    # math
```

Comment:
Move this `# math` section above `# text` to keep sorted in the same fashion as the imports.